examples/pom.xml

```diff
@@ -221,14 +221,21 @@
       <groupId>com.github.cukespace</groupId>
       <artifactId>cukespace-core</artifactId>
       <version>${project.version}</version>
-      <scope>test</scope>
       <exclusions>
         <exclusion>
           <groupId>com.google.guava</groupId>
           <artifactId>guava</artifactId>
         </exclusion>
       </exclusions>
+      <scope>test</scope>
     </dependency>
+    
+    <dependency>
```

Comment:
interesting, which version do you use? I have no issue with 3.0 and 3.2 o fmaven